client/src/views/users/GreetUserModal.vue

```diff
@@ -123,21 +123,21 @@
         <!-- Final Step -->
         <div
           v-show="pageStep === GreetUserStep.Summary"
-          v-if="userInfo"
+          v-if="guestInfoPage"
           class="step final-step"
         >
           <div class="final-step__name">
             <user-avatar-name
-              :user-name="userInfo.name"
-              :user-avatar="userInfo.name"
+              :user-name="guestInfoPage?.fullName"
+              :user-avatar="guestInfoPage?.fullName"
```

Comment:
the same information is passed twice, there should be only one props and the user-avatar-name should use it for both on its own then.

Reply:
Wanted to avoid changing UserAvatarName. It has been designed to be able to take something else as the user avatar, but in the meantime, we use the user name.